pylxd/models/instance.py

```diff
@@ -853,7 +853,7 @@ def publish(self, public=False, wait=False):
         If wait=True, an Image is returned.
 
         This functionality is currently broken in LXD. Please see
-        https://github.com/lxc/lxd/issues/2201 - The implementation
+        https://github.com/canonical/lxd/issues/2201 - The implementation
```

Comment:
Looks like this one has also been fixed.

Reply:
Same here